src/net.cpp

```diff
@@ -970,7 +970,6 @@ void CConnman::AcceptConnection(const ListenSocket& hListenSocket) {
     CAddress addr;
     int nInbound = 0;
     int nMaxInbound = nMaxConnections - (nMaxOutbound + nMaxFeeler);
```

Comment:
I don't think a negative value for something called `nMaxOutbound` makes sense. There should never be a difference in behavior between no outgoing connections, or negative outgoing connections. But ok, this needs a different solution anyway at a higher level.